query/src/storages/fuse/meta/common.rs

```diff
@@ -34,6 +34,7 @@ pub struct Statistics {
     pub compressed_byte_size: u64,
 
     pub col_stats: HashMap<ColumnId, ColumnStatistics>,
+    pub cluster_stats: HashMap<ColumnId, ColumnStatistics>,
```

Comment:
Just a reminder:

we need a `#[serde(default)]` attr for the new field `cluster_stats` to keep backward compatible in the final PR